code/game/objects/items/storage/bags.dm

```diff
@@ -162,6 +162,26 @@
 /obj/item/storage/bag/ore/cyborg
 	name = "cyborg mining satchel"
 
+/obj/item/storage/bag/ore/cyborg/ComponentInitialize()
+	. = ..()
+	GET_COMPONENT(STR, /datum/component/storage/concrete/stack)
+	STR.allow_quick_empty = TRUE
+	STR.can_hold = typecacheof(list(/obj/item/stack/ore))
+	STR.max_w_class = INFINITY
+	STR.max_combined_stack_amount = 150
+
+/obj/item/storage/bag/ore/large
+	name = "large mining satchel"
+	desc = "This bag can hold three times the ore in many small pockets. Shocking folding and compact for its volume."
+
+/obj/item/storage/bag/ore/large/ComponentInitialize()

```

Comment:
no, you only need to edit max_combined_stack_amount

Reply:
kk